### PR TITLE
Add ptb tutorial

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
Related to #2576
Write the word2vec tutorial by writing the explanation of chainer/examples/word2vec/.

Although it is in WIP, please advice me to improve the tutorial.

### CODE REVIEW

File: docs/source/tutorial/ptb.rst [hunk 1]
```diff
@@ -0,0 +1,293 @@
+The Implementation of Recurrent Neural Net Language Model with Chainer
```
Comment (mitmul) [inline docs/source/tutorial/ptb.rst]: "with Chainer" is obvious, so please omit it. Then how about "Write an RNN Language Model" for the title?

**Comment by mitmul** [comment]
I added some explanations and fixed some descriptions. Submitting those changes as review comments ins inefficient way, so I wrote the modified version here: https://github.com/mitmul/chainer-notebooks/blob/master/4_RNN-language-model.ipynb

All the code have been checked and those work fine. You can see the outputs of the code in the above jupyter notebook.

**Comment by keisuke-umezawa** [comment]
I finished fixing.

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by mitmul** [comment]
@keisuke-umezawa The doctest is failing due to this error:
```
17:06:05 Warning, treated as error:
17:06:05 /work/chainer/docs/source/tutorial/ptb.rst:: WARNING: document isn't included in any toctree
```
Could you check it and fix?

**Comment by keisuke-umezawa** [comment]
@mitmul
I fixed the warning. Please retry the test.

**Comment by keisuke-umezawa** [comment]
Jenkins, test this please

**Comment by keisuke-umezawa** [comment]
It seems I do not have the right to kick the CI...

**Comment by keisuke-umezawa** [comment]
Jenkins, test this please

**Comment by mitmul** [comment]
LGTM

**Pull request merged**